CHANGELOG.md

```diff
@@ -2,10 +2,12 @@
 
 ## Unreleased changes
 
-New features:
+Spotlight change:
 
 - ParaTime withdrawals now automatically set the fee that will soon be required on Emerald
   (0.00015 ROSE/TEST)
+- Warn users on ParaTime withdraws (and deposits) to accounts they don't own
+  ([#236](https://github.com/oasisprotocol/oasis-wallet-ext/pull/236)).
```

Comment:
I rebased and added this ^

Reply:
thanks!

Reply:
I'm going to try moving this into a separate PR, where we're disabling the warning on deposit